### ISSUE TITLE
Feature request: cut only

### ISSUE THREAD

**Issue by ZdenekM** [pklaus/brother_ql#74]
Would it be possible to add command to perform only cut? Like if I print label without cutting and then need to cut it later.

**Comment by Mikhail5555** [comment]
You could theoretically feed it a full white image, or use the cut button on your label printer?

**Comment by ZdenekM** [comment]
I need to pick the printed label with a robot (one arm only) so pressing a cut button is not an option unfortunately :-) I tried to print full white image with 1px length but probably due to some margins or whatever, the printer moves out approximately 2 cm of label and then cuts. If I print and cut, the printed label is 'fired' from the printer so I need to print it, grasp using a robot's suction cup and then cut it.

**Comment by Mikhail5555** [comment]
You could possibly try removing the line that adds the magin to the printing payload (https://github.com/pklaus/brother_ql/blob/master/brother_ql/conversion.py#L188), that might lead to it 'printing' only the 1px and then cutting it.

**Comment by Mikhail5555** [comment]
I did quite a bit of googling, since this was quite an interesting suggestion, sadly I didn't get a very great result.
TLDR version: This library uses Raster mode to print it on the printers. They also have an ESC/P mode (which only some printers support from my research) which does state that there is a printer operation to 'Only cut'.  (See screenshot below, taken from the developer documentations of the QL-720NW printer), So maybe just maybe it would work, but even then, it look like it still feeds a minimum amount before it cuts.

![Screenshot-000_18-01-2020](https://user-images.githubusercontent.com/7824786/72655837-86a49780-3997-11ea-8e1d-92db8a0f0b93.png)

**Comment by ZdenekM** [comment]
@Mikhail5555 Thank you for your effort. As a first attempt, I will try to remove margins.
